keyboards/bastardkb/dilemma/4x6_4/4x6_4.c

```diff
@@ -57,3 +57,59 @@ bool encoder_update_kb(uint8_t index, bool clockwise) {
     return true;
 }
 #endif // ENCODER_ENABLE
+
+#ifdef RGB_MATRIX_ENABLE
+// Layer state indicator
+bool rgb_matrix_indicators_advanced_kb(uint8_t led_min, uint8_t led_max) {
+    if (!rgb_matrix_indicators_advanced_user(led_min, led_max)) { return false; }
+    if (host_keyboard_led_state().caps_lock) {
+        for (int i = led_min; i <= led_max; i++) {
```

Comment:
`const int i = …`

Reply:
Results in `error: increment of read-only variable 'i'` when it's incremented in the loop's step instruction.
Did I misunderstand your request?

Reply:
Yeah, I'm dumb, wrong language.